kafka_exporter.go

```diff
@@ -680,6 +689,12 @@ func main() {
 		[]string{"consumergroup"}, labels,
 	)
 
+	customLabelsEnv, isSet := os.LookupEnv("CONSUMERGROUP_LAG_CUSTOM_LABELS")
+	if isSet{
+		plog.Infoln("Env CONSUMERGROUP_LAG_CUSTOM_LABELS is set, loding custom owner label for consumergroups")
+		customLabels, _ = NewCustomCGLagLabels(customLabelsEnv)
```

Comment:
should we do something with the error here? at least logging that this is not going to work

Reply:
done